setup.py

```diff
@@ -34,7 +34,7 @@
         'Programming Language :: Python :: 3.7'
         ],
       packages=find_packages(),
-      install_requires=['indra', 'boto3', 'jsonpickle', 'kappy==4.0.94',
+      install_requires=['indra', 'boto3', 'jsonpickle', 'kappy==4.1.2',
```

Comment:
We may have to revert this one due to unforeseen issues, but would also have to make changes in the Dockerfile to properly install the older version.